src/webui/master/static/home.html

```diff
@@ -162,14 +162,15 @@
           <th data-key="name">Task Name</th>
```

Comment:
is this task id (MESOS_TASK_ID?), should this be "Task Id"?

Reply:
Task ID is one line above